tests/tensor_search/test_validation.py

```diff
@@ -301,6 +304,64 @@ def test_boost_validation_None_ok(self):
             assert None is validation.validate_boost(boost=None, search_method=search_method)
 
 
+class TestValidateSearchableAttributes(unittest.TestCase):
+    
+    def setUp(self) -> None:
+        self.field_names = [f"field{i}" for i in range(10)]
+        self.searchable_attributes = [f"field{i}" for i in range(5)]
+
+    def test_search_method_not_tensor(self):
+        validation.validate_searchable_attributes(
+            self.field_names,
+            self.searchable_attributes,
+            search_method=enums.SearchMethod.LEXICAL
+        )
+
+    def test_maximum_searchable_attributes_not_set(self):
+        validation.validate_searchable_attributes(
+            self.field_names,
+            self.searchable_attributes,
+            search_method=enums.SearchMethod.TENSOR
+        )
+
+    def test_searchable_attributes_is_none_max_value_set_raise_invalid_arg_error(self):
+        try:
+            validation.validate_searchable_attributes(
+                self.field_names,
+                searchable_attributes=None,
+                search_method=enums.SearchMethod.TENSOR
+            )
```

Comment:
insert raising an assertion error here - this ensure that the test fails if the expected error is not raised

Reply:
also, consider `environ` here too, to make this test more deterministic

Reply:
Smart.